script.js

```diff
@@ -45,32 +46,38 @@ function startup() {
 			video.play();
 		})
 		.catch(function (err) {
-			console.log('An error occurred: ' + err);
+			console.log("An error occurred: " + err);
 		});
 
 	video.addEventListener(
-		'canplay',
+		"canplay",
```

Comment:
please donot change single quotes to double.

Reply:
It change due to extension of vs code (prettier).While formatting file